### ISSUE TITLE
SQLite Get ExpiresAt is {1/1/0001 12:00:00 AM +00:00}

### ISSUE THREAD

**Issue by VictorioBerra** [dotnetcore/EasyCaching#97]
## Description

![image](https://user-images.githubusercontent.com/2934507/54468685-32b89900-475d-11e9-90a7-690e9f571a37.png)

### Steps to Reproduce

1. SetAsync
2. Refresh
3. GetAsync

**Expected behavior:** [What you expected to happen]
An expiration > 1/1/0001

**Actual behavior:** [What actually happened]
expiration == 1/1/0001

## Specifications

- Provider    : SQLite (version 0.5.3)

**Comment by catcherwong** [comment]
@VictorioBerra  Thanks for your interest in this project.

I feel so sorry for this mistake! 

`ExpiresAt` was added when I prepared to implement mutex key for Get/GetAsync(with data retriever) method. 

But I use a simple way to do and forget to remove `ExpiresAt` in `CacheValue<T>`.

The real expiration of a key when using SQLite was stored in a database.

https://github.com/dotnetcore/EasyCaching/blob/2f46b2eaac00935a1b42beb84bb765730aaad1d1/src/EasyCaching.SQLite/Configurations/ConstSQL.cs#L11-L22

At current time, EasyCaching don't support to get a key's TTL.
